test-suite/Makefile

```diff
@@ -39,8 +39,7 @@ BIN := $(shell cd ..; pwd)/bin/
 COQFLAGS?=
 
 ifeq ($(COQLIB),)
```

Comment:
This variable is used before being defined, it seems.

> Makefile:41: warning: undefined variable `COQLIB'

What about adding a line like `COQLIB?=` before?

Reply:
Sure, I'll add that.  make doesn't generate such warnings on my system.  Perhaps you're picking up some additional command-line settings?

Reply:
I’m trying to understand the failures of the CI. In both cases, I can see this warning that I find suspicious.

Reply:
... though I do see that message in the CI output.

Reply:
Hmm, that warning should also have appeared without the changes of this PR.